docs/queries/sorting.md

```diff
@@ -36,6 +37,7 @@ You can sort tasks by the following properties:
 5. `done` (the date when the task was done)
 6. `path` (the path to the file that contains the task)
 7. `description` (the description of the task)
+8. `tag` (the description of the task)
```

Comment:
Should this be `tags` and is `the description of the task` a copy-paste slip from the previous line?